index.html

```diff
@@ -215,13 +311,14 @@ <h3 class="language">Skills</h3>
               <li>Gitlab</li>
               <li>Terminal</li>
             </ul>
+        </section>
       </section>
     </main>
   
     <footer> 
       <section class="contact">
         <p>I'm always interested in hearing about new projects,
-        so if you'd like to chat please get in</p>
+        so if you'd like to chat please get in touch</p>
         <form action="https://formspree.io/f/xjvzpjgq" method="post">
           <input type="text" name="fullname" placeholder="Full Name" maxlength="30" required>
           <input type="email" name="email" placeholder="Email address" required>
```

Comment:
- [ ] Please increase the width of the line and make it 100%. So it matches the design
### Your design
![ssssssssssss](https://user-images.githubusercontent.com/81727988/199898570-8da8e60d-9dda-48db-a6bf-d32c810b492d.png)